specs-go/config.go

```diff
@@ -90,8 +98,10 @@ type Mount struct {
 	// Source specifies the source path of the mount.  In the case of bind mounts on
 	// Linux based systems this would be the file on the host.
 	Source string `json:"source"`
-	// Options are fstab style mount options.
-	Options []string `json:"options,omitempty"`
+	// Options are fstab style mount options. (this field is platform dependent)
+	Options []string `json:"options,omitempty" platform:"linux,solaris"`
+	// Readonly specifies if the mount should be read-only. (this field is platform dependent)
+	Readonly bool `json:"readonly" platform:"windows"`
```

Comment:
Could you parse a constant `readonly` from existing `options` to set this boolean for Windows?